content/articles/2017-12-14-styler-1-0-0.Rmd

```diff
@@ -0,0 +1,230 @@
+---
+title: styler 1.0.0
+author: Kirill Müller, Lorenz Walthert
+date: '2017-12-14'
+slug: styler-1.0.0
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/mpdIPhYqZ4Y
+  author: Heng Films
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE, comment = "#>")
+knitr::knit_engines$set(list(
+  styler = function(options) {
+    options$comment <- ""
+    knitr::engine_output(
+      options,
+      
+      c("# Before", options$code),
+      c("# After", styler::style_text(options$code))
+    )
+  }
+))
+
+```
+
+
+We're pleased to announce the release of [styler 1.0.0](https://github.com/r-lib/styler/releases/tag/v1.0.0). styler is a source code formatter - a package to format R code according to a style guide. We've implemented the [tidyverse style guide](http://style.tidyverse.org), so you can format your code according to it. A coherent style is important for consistency and legibility. Just as it is important to putspcesbetweenwords. 
+You can install styler from CRAN.
+
+```{r, eval = FALSE}
+install.packages("styler")
+```
+
+styler can style text, single files, packages and entire
+R source trees with the following functions:
+
+* `style_text()` styles a character vector.
+* `style_file()` styles R and Rmd files.
+* `style_dir()` styles all R and/or Rmd files in a directory.
+* `style_pkg()` styles the source files of an R package.
+* An RStudio Addin that styles the active file R or Rmd file, the current 
+  package or the highlighted code.
+
+## styling options
+
+A distinguishing feature of styler is its flexibility. The following options are available:
+
+* `scope`
```

Comment:
Maybe add a few words to outline what each of the options means?

Reply:
Ok.